### PR TITLE
Add taxonomy test

### PR DESCRIPTION
**Pull request by cerritows**
## Description
- Add test scenario for taxonomy class
- Fix variable initialization
- Add to ´init´

### CODE REVIEW

File: test/taxonomy.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from factiva.news import Taxonomy
+
+taxonomy = Taxonomy()
+print(taxonomy.categories)
+industries_code = taxonomy.get_codes('industries')
+print(industries_code)
+company = taxonomy.get_company('isin', company_code='PLUNMST00014')
+code_list = ["US0378331005", "US0231351067", "US5949181045", "US4523531083"]
```
Comment (vic3r) [inline test/taxonomy.py:8]: code_list is not being used

File: test/taxonomy.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from factiva.news import Taxonomy
+
+taxonomy = Taxonomy()
+print(taxonomy.categories)
+industries_code = taxonomy.get_codes('industries')
+print(industries_code)
+company = taxonomy.get_company('isin', company_code='PLUNMST00014')
```
Comment (vic3r) [inline test/taxonomy.py:7]: I was thinking, may the code_type (first argument) could be a kwarg like the company_code, in order to be more friendly

**Review by vic3r — CHANGES_REQUESTED** [review]
the code_list is not being used

File: test/taxonomy.py (hunk 2 above)
Comment (cerritows) [reply test/taxonomy.py:7]: I thought about it. It would look better, however the way to call the method would stay pretty much the same.

**Review by vic3r — APPROVED** [review]
👍

**Pull request merged**